Formula/profanity.rb

```diff
@@ -29,13 +31,18 @@ class Profanity < Formula
   depends_on "libotr" => :recommended
   depends_on "gpgme" => :recommended
   depends_on "terminal-notifier" => :optional
+  depends_on :python if build.with? "python-plugin-support"
```

Comment:
Sorry, I'm getting a little confused, so let me summarize …

I found the (original) homebrew formula of profanity comes with `--disable-python-plugins` configure option by default and so comes the bottled version.

Compiling profanity against python is needed if - and only if - python plugin support is desired.

Using `depends_on :python if build.with? "python-plugin-support` (i.e. the symbol `:python`) homebrew will automatically select either the system python (if no homebrew python is installed) or the homebrewed python if it is installed - what seems coherent to me (since I didn't want to change the default behaviour of the original formula).

It will compile against both (system and homebrew python) - at least on macOS sierra (couldn't test it on older systems …)